src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = {
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall()
+        };
+
+        for (Ball ball: balls) {
```

Comment:
```suggestion
        for (Ball ball : balls) {
```

Reply:
use Intellij shortcut to auto-format before committing - like Ctrl Alt L if you have Windows